sal_interfaces/ATDome/ATDome_Commands.xml

```diff
@@ -33,12 +33,12 @@
         <Action/>
         <Value/>
         <Description>Close both shutter doors. This is the preferred way to close the shutter, because it automatically sequences the doors to prevent the main door from interfering with the dropout door.</Description>
-        <Explanation>http://sal.lsst.org</Explanation>
+        <Explanation/>
         <item>
             <EFDB_Name>ignored</EFDB_Name>
             <Description/>
```

Comment:
https://ts-xml.lsst.io/sal_constraints_and_recommendations.html#sal-topic-and-parameter-description-and-unit-fields
>XML will have the <Units> and <Description> fields defined for each parameter in a topic

We are missing the description for the parameters

Reply:
Please consider relaxing this rule for a topic that has one field. The description of the field is redundant with the description of the topic. That said, I hear and obey. In some cases the results are silly but in others they were an improvement. ("active" is a strange name for a field that means `communicating` but I doubt it's worth changing).

Reply:
(On the other hand I guess that would make automatic checking a bit more complicated.)

Reply:
I know the redundancy is kind of a pain, but the completeness is more important - plus you are right, the testing will be more problematic.   So a quick cut and paste by the developer shouldn't be too much effort.